server/routes/reviewPlan/reviewCheckYourAnswersController.ts

```diff
@@ -1,11 +1,18 @@
 import { RequestHandler } from 'express'
 import ReviewCheckYourAnswersView from './reviewCheckYourAnswersView'
+import { getPrisonerContext } from '../../data/session/prisonerContexts'
+import { addCurrentPageToHistory } from '../pageFlowHistory'
 
 export default class ReviewCheckYourAnswersController {
   getReviewCheckYourAnswersView: RequestHandler = async (req, res, next): Promise<void> => {
+    const { prisonNumber } = req.params
     const { prisonerSummary } = res.locals
+    const userName = res.locals.user.name
+    addCurrentPageToHistory(req)
```

Comment:
Add the check your answers page to history to enable navigation from the review or notes pages if they've been reached from a Change link.